product_docs/docs/postgres_distributed_for_kubernetes/1/known_issues.mdx

```diff
@@ -14,15 +14,15 @@ This version of EDB Postgres Distributed for Kubernetes **doesn't support** migr
 
 ## Connectivity with PgBouncer
 
-EDB Postgres Distributed for Kubernetes does not support using [PgBouncer](/pgbouncer/latest/) to pool client connection requests. 
-This applies to both the open-source and EDB versions of PgBouncer.
+EDB Postgres Distributed for Kubernetes **doesn't support** using [PgBouncer](/pgbouncer/latest/) to pool client connection requests. 
```

Comment:
To be consistent with your other items. (I ordinarily wouldn't bold text.)

Reply:
I agree with remaining consistent with the usage of bold text. 

And I think this might be one of the instances where want to not use contractions. 
"Does not" is a bit stronger than "doesn't". 
Also, I believe Release Notes and Known Issues should be more formal and less conversational. 

```suggestion
EDB Postgres Distributed for Kubernetes **does not support** using [PgBouncer](/pgbouncer/latest/) to pool client connection requests. 
```

PS: adding as item to discuss in the Style Guide Counsil.